src/utils.ts

```diff
@@ -81,3 +81,15 @@ export const networkFromId = (id: string) => {
       return null
   }
 }
+
+export const networkFromString = (name: string) => {
+  switch (name) {
+    case 'mainnet':
+    case 'main':
```

Comment:
you can move these two cases down to the bottom, above `default`, to avoid the redundant Network.Main

Reply:
@alexanderatallah done 👍